### PR TITLE
Blockchain configuration page ui

### PR DESCRIPTION
**Pull request by nicholasmueller**
Issue/Task Number: https://github.com/omisego/ewallet/issues/1119

https://gallery.io/projects/MCHbtQVoQ2HCZfyN0ITInFtb/files/MCEJu8Y2hyDScSavJ4fVcnij3YETz93IPwQ

Moved each configuration setting to separate pages
Blockchain settings is just placeholder for now, not hooked up yet, also blockchain connection status is dummy placeholder for now

### CODE REVIEW

**Comment by jarindr** [comment]
Screen Shot 2562-07-21 at 19 37 17
on file storage setting, when choosing an options it breaks the page 😆

File: apps/frontend/assets/src/omg-page-configuration/CacheSettings.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+`
+const Grid = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+`
+
+const CacheSettings = (props) => {
+  useEffect(() => {
+    return props.handleCancelClick
```
Comment (jarindr) [inline apps/frontend/assets/src/omg-page-configuration/CacheSettings.js]: you're being more and more functional now lol

**Comment by nicholasmueller** [comment]
@jarindr made fixes

**Comment by jarindr** [comment]
👍 merging

**Pull request merged**